### ISSUE TITLE
MANA Balance not updating correctly

### ISSUE THREAD

**Issue by Tudamoon** [decentraland/builder#2395]
## 🎉 Description
![image](https://user-images.githubusercontent.com/111532622/199392534-c42ab912-bd6f-4abb-b32a-4d70412bcdea.png)
![image](https://user-images.githubusercontent.com/111532622/199392632-9b788503-1187-48b3-a80e-aa64c8ac200f.png)
I am trying to publish a wearable collection and my MANA balance is updated on the Ethereum Mainnet as well as on the top right corner of the builder. However when I go to publish my collection it shows my old balance in the Modal. 

* [ x] 🐛 This is a bug report.
* [ ] 📈 This is a feature request.

## 📝 Details

I am trying to publish a wearable collection and my MANA balance is updated on the Ethereum Mainnet as well as on the top right corner of the builder. However when I go to publish my collection it shows my old balance in the Modal. It will not let go further than the modal. 

I am using the Brave Browser on Windows 10. I have taken numerous steps to try to solve this. I have cleared my cookies and cache on my browser. I have disconnected my wallet and reconnected it. I did a hard refresh to the page. I even switched from my desktop to my laptop to try to fix the issue. I am pretty certain there is an issue with updating the MANA balances.

## 🔢 Steps To Reproduce Issue [ Good To Have ]

Please remember that with baby steps it's easier to reproduce the bug and it's much faster to fix it.

```
- Step 1 Go to builder.decentraland.org
- Step 2 Go to Collections tab
- Step 3 Go to collection that is ready to publish
- Step 4 Attempt to publish collection and it will show older previous balance of MANA on the Ethereum Mainnet.
...
```

**Comment by juanmahidalgo** [comment]
Hi @Tudamoon, I'm sorry you're having trouble when publishing your collection.
He problem is that you don't have enough MANA in the Polygon network (the one that is used for covering the publishing fees). Please use our Account dApp (https://account.decentraland.org) to bridge your Ethereum MANA to Polygon so you have enough balance to cover it.

**Comment by Tudamoon** [comment]
The wearable curation actually requires MANA to be on the Ethereum Network.

**Comment by Tudamoon** [comment]
The symbols on the Builder submission form is incorrect also.

**Comment by cazala** [comment]
The curation committee requires MANA on Polygon. The white diamond is Polygon MANA, the red hexagon is Ethereum MANA. You can move MANA between Ethereum and Polygon using the [Account dApp](https://account.decentraland.org)

**Comment by Tudamoon** [comment]
That is incorrect. I understand the White Diamond is Polygon MANA, the Red Hexagon is Ethereum MANA, however, it pulls funds from the Ethereum MANA. This is not my first time submitting a wearable. Please look at your code where it pulls the MANA from. Your code pulls MANA from the Ethereum Network. So with that being said, either you change the symbols shown when you try to publish the selection or you change the network it pulls the MANA from.

**Comment by cazala** [comment]
If you already published a wearable or an emote, look for the transaction, you will find it on Polygon. User generated wearables exist only on Polygon and they have alreays required Polygon MANA for the publication fee. It is actually not possible to make a Polygon transaction to move your funds on Ethereum.
